src/inmanta/data/__init__.py

```diff
@@ -2691,8 +2700,10 @@ async def get_released_resources(
             end=end,
             **query,
         )
-
-        db_query = f"SELECT * " f"{cls._get_released_resources_partial_base_query()}"
+        db_query, base_query_values = cls._get_released_resources_base_query(
+            select_clause="SELECT * ", environment_filter=query["environment"], offset=len(values) + 1
```

Comment:
if environment is always assume to be present, I would not hide it in the kwargs but make it a proper argument.